src/canvas/StaticCanvas.ts

```diff
@@ -65,6 +65,25 @@ type TCanvasHydrationOption = {
   signal?: AbortSignal;
 };
 
+export const StaticCanvasDefaults = {
+  backgroundColor: '',
+  backgroundImage: null,
+  overlayColor: '',
+  overlayImage: null,
```

Comment:
are the blank values needed?

Reply:
i think because are either string or gradients, and TS bother us then if are undefined.
I won't touch any default value in this PR, since this PR is just moving them.
I can experiment right away after